docs/src/assets/scss/components/rules.scss

```diff
@@ -90,6 +90,16 @@
     flex: 1 1 35ch;
 }
 
+.rule__name_wrapper {
+    display: flex;
+    align-items: center;
+    gap: 0.5rem;
+
+    .frozen {
+        font-size: 12px;
+    }
```

Comment:
I feel we can increase it's size a bit to make it more noticeable?
right now
![Screenshot 2024-12-17 191833](https://github.com/user-attachments/assets/5c8f4cfe-9588-496f-ac6b-f14c06c4e430)
we can increase it to `.875rem` as other icons.
![Screenshot 2024-12-17 191947](https://github.com/user-attachments/assets/81fc437a-ac99-45c4-b662-478d0018a6ce)
or to `1rem`
![Screenshot 2024-12-17 192013](https://github.com/user-attachments/assets/4caecf11-22cd-4d3b-9f45-1826931dc030)

Reply:
ya, we can set  `.875rem` it looks better.